lib/vpqr.js

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+const cborld = require('@digitalbazaar/cborld');
+const qrcode = require('@nuintun/qrcode');
+import citContext from 'cit-context';
+
+import base32Encode from 'base32-encode';
+import base32Decode from 'base32-decode';
+
+const {
+  Encoder,
+  // QRByte,
+  QRAlphanumeric,
+  // ErrorCorrectionLevel
+} = qrcode;
+
+const {appContextMap: citAppContextMap} = citContext;
+
+const VP_QR_VERSION = 'VP1';
+const BASE_32_UPPERCASE_MULTIBASE_PREFIX = 'B';
+
+function _bytesToQrCodeDataUrl({bytes, size}) {
+  const qrcode = new Encoder();
+  qrcode.setEncodingHint(true);
+  // qrcode.setErrorCorrectionLevel(ErrorCorrectionLevel.M);
+  const encoded = base32Encode(bytes.buffer, 'RFC4648', {padding: false});
```

Comment:
The one thing I don't like about `https://github.com/LinusU/base32-encode` is that it uses an `ArrayBuffer` as the input ... what if your `Uint8Array` (a view on an `ArrayBuffer`) did not span the entire buffer? We can ignore this at this very moment, but it seems like it could become trouble. I'll file an issue to get it fixed later.

Reply:
Addressed via new upstream version (issue #2).